### ISSUE TITLE
Add check for float(sys.version_info.major)

### ISSUE THREAD

**Issue by MarcoGorelli** [asottile/flake8-2020#52]
I got this in a PR today:
```python
python_version = float(sys.version_info.major) + 0.1 * float(
    sys.version_info.minor,
)
```

`flake8-2020` doesn't catch this, would it be a welcome addition?

**Comment by asottile** [comment]
closing => archiving
